plugins/module_utils/zos_mvs_raw.py

```diff
@@ -99,3 +129,308 @@ def __init__(self, rc, stdout, stderr):
+            block_size : int, optional
+                       The block size of the data set. Defaults to None.
+            directory_blocks : int, optional
+                             The number of directory blocks to allocate for the data set. Defaults to None.
+            record_format : str, optional
+                          The record format of the data set. Defaults to None.
+            record_length : int, optional
+                          The length, in bytes, of each record in the data set. Defaults to None.
+            sms_storage_class : str, optional
+                              The storage class for an SMS-managed dataset. Defaults to None.
+            sms_data_class : str, optional
+                           The data class for an SMS-managed dataset. Defaults to None.
+            sms_management_class : str, optional
+                                 The management class for an SMS-managed dataset. Defaults to None.
+            key_length : int, optional
+                       The key length of a record. Defaults to None.
+            key_offset : int, optional
+                       The key offset is the position of the first byte of the key
+                       in each logical record of a the specified VSAM data set. Defaults to None.
+            volumes : list, optional
+                    A list of volume serials.. Defaults to None.
+            key_label : str, optional
+                      The label for the encryption key used by the system to encrypt the data set. Defaults to None.
+            encryption_key_1 : dict, optional
+                             [description]. Defaults to None.
+            encryption_key_2 : dict, optional
+                             [description]. Defaults to None.
+            reuse : bool, optional
+                  Determines if data set should be reused. Defaults to None.
+            replace : bool, optional
+                    Determines if data set should be replaced. Defaults to None.
+            backup : bool, optional
+                   Determines if a backup should be made of existing data set when disposition=NEW, replace=true,
+                   and a data set with the desired name is found.. Defaults to None.
+            return_content : dict, optional
+                           Determines how content should be returned to the user. Defaults to None.
+            tmphlq : str, optional
+                   HLQ to be used for temporary datasets. Defaults to None.
+        ----------
+        """
+        self.backup = None
+        self.return_content = ReturnContent(**(return_content or {}))
+        self.tmphlq = tmphlq
+        primary_unit = space_type
+        secondary_unit = space_type
+        key_label1 = None
+        key_encoding1 = None
+        key_label2 = None
+        key_encoding2 = None
+        if encryption_key_1:
```

Comment:
Not sure of what is the use of encryption_key_1 and encryption_key_2. Could you explain? docstring  does provide description for other attributes but not for this one.

Reply:
https://github.com/ansible-collections/ibm_zos_core/blob/b16ceb62643f8da42535100f6f68b9271758b93f/plugins/module_utils/dd_statement.py#L340
Here is the description.